framework/src/modules/interoperability/utils.ts

```diff
@@ -832,3 +833,16 @@ export const chainAccountToJSON = (chainAccount: ChainAccount) => {
 		status,
 	};
 };
+
+export const getMainchainID = (chainID: Buffer): Buffer => {
```

Comment:
maybe we can make this as constant `Buffer.alloc(CHAIN_ID_LENGTH - 1, 0)` and use it across the system as this will be a protocol constant

```ts
return Buffer.concat([networkID, MAINCHAIN_ID_WITHOUT_NETWORK_PREFIX]);
```

Reply:
or maybe we can always use interops method to compute the `mainchainID` so it doesn't exist in other places

Reply:
Created issue https://github.com/LiskHQ/lisk-sdk/issues/7780